Ubuntu_Bionic_Install.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+export NODEJS_INCLUDE_DIRS=$HOME/.nvm/versions/node/v10.15.0/include
+
+sudo apt update
+sudo apt upgrade -y
+
+sudo apt install -y mc
+
+sudo apt install -y cmake make libtool pkg-config g++ gcc jq lcov protobuf-compiler vim-common libboost-all-dev libboost-all-dev libcurl4-openssl-dev zlib1g-dev liblz4-dev libprotobuf-dev
+
+
+#if you plan to compile with data building support, see below for more info
+
+sudo apt install -y libgeos-dev libgeos++-dev liblua5.2-dev libspatialite-dev libsqlite3-dev lua5.2 wget
+if [[ $(grep -cF bionic /etc/lsb-release) > 0 ]]; then sudo apt install -y libsqlite3-mod-spatialite; fi
+
+
+#if you plan to compile with python bindings, see beloNODEJS_INCLUDE_DIRSw for more info
```

Comment:
seems to be a typo here?

Reply:
Of course it is... kind of lost ctrl+v :)